### ISSUE TITLE
fastdup.run() saves temp files in root path

### ISSUE THREAD

**Issue by davertor** [visual-layer/fastdup#37]
I executed the following command

```
fastdup.run(input_dir=train_imgs_list,
            work_dir=output_folder,
            test_dir=test_imgs_list,)
```

Despite I set work_dir output folder, the code saves files as .sentry-native/, files.txt and testfiles.txt in the root path

**Comment by dbickson** [comment]
Hi @davertor thanks for your report. We are working on a fix to be ready today in version 0.165. We will update once it is ready.

**Comment by dbickson** [comment]
HI @davertor it should be fixed now can you try again on version 0.165 (for ubuntu 18, 20) we will compile a version for mac later today.

**Comment by dbickson** [comment]
Should be fixed now on version 0.165.

**Comment by Tompil3r** [comment]
issue still occurring with files.txt

**Comment by dbickson** [comment]
Should be fixed in 0.170
